### ISSUE TITLE
Possible Underestimated pass@3 Results

### ISSUE THREAD

**Issue by aorwall** [nus-apr/auto-code-rover#45]
I have evaluated your predictions using my [Docker based swe-bench evaluator](https://github.com/aorwall/SWE-bench-docker/tree/main). I achieve 26% on pass@3 compared to the 22% you reported. It might be worthwhile to review the logs for the failed benchmarks to see if your agent can actually achieve even better results :D

**Comment by zhiyufan** [comment]
Thank you for this detailed report! 
I quickly went through the comparison sheet you shared and noticed that there are many opposite acr-2/3 results in the two evaluation environments. Maybe you mistakenly mixed up the acr-2/3 results?

**Comment by aorwall** [comment]
You're right. Good catch. That explains why it looked like some of the benchmark instances failed in my benchmark. Now it covers all the cases you resolved.

**Comment by yuntongzhang** [comment]
This looks great! The dockerized evaluation environment would be very useful for obtaining consistent evaluation results. We will also try out your docker-based evaluation on our machines :)
